hera_sim/foregrounds.py

```diff
@@ -118,7 +117,7 @@ def pntsrc_foreground(lsts, fqs, bl_vec, nsrcs=1000, Smin=0.3, Smax=300,
         dtau = np.random.uniform(-.1 * bl_len_ns, .1 * bl_len_ns)  # XXX adds a bit to total delay, increasing bl_len_ns
         vis[t, :] += flux * (fqs / mfreq) ** index * np.exp(2j * np.pi * fqs * dtau)
     ha = utils.compute_ha(lsts, 0)
-    for fi in xrange(fqs.size):
+    for fi in range(fqs.size):
```

Comment:
I'd recommend using a forwards compatibility on range in python 2. something like: 
`from builtins import range`
which brings the python 3 `range` feature into python 2. In py2 `range` returns a list which can cause issues if your iterator is very large. `six.moves` also has a range import too, but I'm not sure what is listed as a dependency for this repo.

Reply:
Fixed this in a67c4f71c5b8bb38be98f811fb70e890acb2402f